### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mebates**
Removed 'in progress' disclaimer
Updated intro section to reflect latest Launch Announcement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,3 +101,7 @@ with this waiver of copyright interest.
 ## Current Sitemap
 
 ![sitemap image](https://raw2.github.com/dezzie/design-manual/gh-pages/assets/img/design_manual_sitemap.png)
+
+## Roadmap
+In the near future we plan to add guidelines for pagination, filtering, modals, tooltips, tabs, and responsive tables. We also plan to make the Design Manual itself fully responsive for small screens. The Design Manual will eventually merge with our existing cfpb.github.io page.
```
Comment (Scotchester) [inline README.md]: Did you want to update/remove that last line, per @iuloshi's comment?

Comment (mebates) [inline README.md]: Or modify? Any suggestions?

Comment (Scotchester) [inline README.md]: How's this?

"The Design Manual will also eventually be given a permanent link in the main navigation of [CFPB Open Tech](https://cfpb.github.io/)."

Comment (mebates) [inline README.md]: Better, but it seems too insignificant to note. Should we just delete?

Comment (Scotchester) [inline README.md]: Yeah, that's fine with me.

**Pull request merged**